easybuild/toolchains/compiler/clang.py

```diff
@@ -84,14 +84,12 @@ class Clang(Compiler):
 
     # used when 'optarch' toolchain option is enabled (and --optarch is not specified)
     COMPILER_OPTIMAL_ARCHITECTURE_OPTION = {
-        systemtools.INTEL : 'march=native',
-        systemtools.AMD : 'march=native',
```

Comment:
oh, wait, this relates to your change in `compiler.py` where you define `self.arch` differently...